guides/common/modules/proc_creating-a-local-file-repository.adoc

```diff
@@ -14,14 +14,25 @@ To create a file type repository in a local directory, complete the following pr
 
 . Ensure the Server and {project-client-name} repositories are enabled.
```

Comment:
This is not correct for upstream. There it's in the pulpcore repositories instead. I'd suggest swapping this line and the next so it's only shown for Satellite.

Reply:
Sent inside `ifdef::satellite[]`